main.c

```diff
@@ -144,11 +144,11 @@ int main(void)
     // fetch own slave address from EEPROM
     uint8_t IdSv = eeprom_read_byte(&EEData[0]);
 
-    #ifdef _SHT31_H
+    #ifdef CALIBRATION
     // fetch Temperature Offset from EEPROM
-    float SHT31_TOffset = eeprom_read_byte(&EEData[1]) / 10.0f;
+    float generic_Temp_Offset = eeprom_read_byte(&EEData[1]) / 10.0f;
```

Comment:
* just a cosmetic nit (not mandatory)
* s/generic_TempOffset/CAL_TOffset/

Reply:
renamed the calibration variables